spec/features/publish_record_spec.rb

```diff
@@ -68,18 +66,8 @@ def stub(url, file_path)
     click_button 'Continue'
 
     expect(page).to have_content 'Check the details of your new country'
-    click_button 'Submit'
```

Comment:
Might be nice to test this approval flow in another scenario? And a basic user submitting a change to be reviewed.
Happy for this to be a separate PR though if it is non-trivial work.

Reply:
Yeah, this is a tough one so a separate PR would be a better idea

Reply:
Agreed but non-trivial :)